public/apps/configuration/panels/audit-logging/audit-logging.tsx

```diff
@@ -64,8 +64,8 @@ function renderStatusPanel(onSwitchChange: () => void, auditLoggingEnabled: bool
               <EuiCode>elasticsearch.yml</EuiCode>. The default storage location is{' '}
               <EuiCode>internal_elasticsearch</EuiCode>, which stores the logs in an index on this
               cluster.{' '}
-              <EuiLink external={true} href="/">
-                Learn more
+              <EuiLink external={true} href={DocLinks.AuditLogsStorageDoc} target="_blank">
+                Learn More
```

Comment:
Is it intended?

Reply:
Copied from landing page. After checking the code base, it has mixed "Learn More" and "Learn more". But the mockup is using "Learn more". Let me do a bulk fix.